Resources/Prototypes/_NF/Roles/Jobs/Nfsd/sheriff.yml

```diff
@@ -26,6 +26,8 @@
   canBeAntag: false
   accessGroups:
   - AllAccess
+  - GeneralNfsdAccess # Frontier
+  - CommandNfsdAccess # Frontier
```

Comment:
``AllAccess``

Reply:
was not able to find AA and figured it was somewhere on the upstream code, and i am trying to keep frontier changes to frontier.  

In addition, not 100% sold on the concept of AA
IMO Sheriff and SR should be able to have their own private rooms or ships.  
AA rejects that concept.  if everybody was still on frontier station them yah, maybe.  but now they have 2 separate stations.  and with more potential command roles coming, i more strongly support the idea of separation of access